tests/conftest.py

```diff
@@ -1,9 +1,13 @@
 import os
 import tempfile
+import sys
 from unittest.mock import MagicMock
 
 import pytest
-import multiprocessing.popen_spawn_posix  # fix for https://github.com/dask/distributed/issues/4168
+
+if sys.platform != "win32":
+    # Fix for https://github.com/dask/distributed/issues/4168
+    import multiprocessing.popen_spawn_posix  # noqa
```

Comment:
Why is this needed in `conftest` as well? Shouldn't the patch in the executor be enough?

Reply:
Because we import the distributed `Client` on the next line